### PR TITLE
Adopt API changes for procps-ng 4.0.5

### PR DESCRIPTION
**Pull request by eworm-de**
The 'info' parameter was dropped in 'VAL' macros.

https://gitlab.com/procps-ng/procps/-/commit/967fdcfb06e20aad0f31073474cb94545c9bdea5

### CODE REVIEW

**Comment by tsujan** [comment]
Thanks!

File: CMakeLists.txt [hunk 1]
```diff
@@ -40,6 +40,9 @@ if (CMAKE_SYSTEM_NAME STREQUAL "Linux")
     message(STATUS "Using PROCPS -> ${PROCPS_MODULE_NAME} v${PROCPS_VERSION}")
     if (PROCPS_VERSION VERSION_GREATER_EQUAL 4.0.0)
         add_definitions("-DUSING_LIBPROC2")
+        if (PROCPS_VERSION VERSION_GREATER_EQUAL 4.0.5)
```
Comment (palinek) [inline CMakeLists.txt:43]: I would avoid next scoping level.
```cmake
    if (PROCPS_VERSION VERSION_GREATER_EQUAL 4.0.5)
        add_definitions("-DUSING_LIBPROC2_1")
    elseif (PROCPS_VERSION VERSION_GREATER_EQUAL 4.0.0)
        add_definitions("-DUSING_LIBPROC2")
    endif()
```

**Comment by tsujan** [comment]
LXQt doesn't start in Arch without this patch anymore. I'll make a point release tonight. Merging…

**Comment by eworm-de** [comment]
Well, Arch already has a fixed package. I am the downstream maintainer for `procps-ng`, thus fixed the issue so I could push the new package. 😜

**Comment by tsujan** [comment]
> Well, Arch already has a fixed package.

Yes, I saw that after adding my comment. But it was necessary to merge your PR for git users and release 2.1.1 for all.

**Comment by eworm-de** [comment]
Yes, of course. This was not a complaint, just a note as Arch was mentioned. We are always happy to drop custom patches, and replace with upstream ones or -even better - with releases. So thanks a lot!

**Comment by tsujan** [comment]
You're welcome, and thank *you* for telling us about it!

**Pull request merged**